### PR TITLE
Check the server_url

### PR DESCRIPTION
**Pull request by bastbnl**

### CODE REVIEW

File: elasticapm/contrib/django/management/commands/elasticapm.py [hunk 1]
```diff
@@ -163,6 +164,38 @@ def handle_check(self, command, **options):
...
+                    )
+                    if ':' in credentials:
+                        self.write('  * SERVER_URL cannot contain authentication '
+                                   'credentials', red, ending='\n')
+                    else:
+                        self.write('  * SERVER_URL contains an unexpected at-sign!'
+                                   ' This is usually used for basic authentication, '
+                                   'but the colon is left out', red, ending='\n')
+                else:
+                    self.write('Server url #lgtm', green)
```
Comment (beniwohli) [inline elasticapm/contrib/django/management/commands/elasticapm.py]: Can you use `SERVER_URL` here, same as in all the other messages? Also, not everybody might be familiar with "lgtm", so maybe just write "`SERVER_URL` looks good"

**Comment by karmi** [comment]
Hi @bastbnl, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/apm-agent-python/pull/226.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

File: elasticapm/contrib/django/management/commands/elasticapm.py (hunk 1 above)
Comment (bastbnl) [reply elasticapm/contrib/django/management/commands/elasticapm.py]: Done. And added testcases

**Comment by bastbnl** [comment]
@karmi Just added and verified the email address. Sorry about that

**Comment by karmi** [comment]
Hi @bastbnl , no worries! Some commits are probably still off, when you use `git commit --reset-author`, it should use any new Git config, you can force push the branch.

**Comment by bastbnl** [comment]
@karmi oh, got to look into the git config force push commands as I'm using pycharm. I'm pretty sure the option is there, but I'll have to search for it. Do you need me to force push with the email address used in the CLA form? 

Happy to create another CLA using the email address used in the commit as that will probably be faster

**Comment by karmi** [comment]
@bastbnl, the [list of commits](https://github.com/elastic/apm-agent-python/pull/226/commits) indicates that you've used two different email addresses in Git, so that would need to be fixed in order for our automatic checks to work... 

I haven't checked the patch content — please follow @beniwohli's suggestions here —, but it seems like the list of commits can be squashed into one, since it's one logical change.

Therefore, please set the [name](https://help.github.com/articles/setting-your-username-in-git/) and [email](https://help.github.com/articles/setting-your-commit-email-address-in-git/) in Git, and run the [interactive rebase](https://help.github.com/articles/about-git-rebase/) on your branch to create a single commit.

Afterwards, use the `git commit --verbose --amend --reset-author` command in terminal, to pick up any new Git configurations for name/email.

The only last step is to: `git push --force`, when the branch is pointed to your origin correctly ("bastbnl:bastbnl-patch-225").

**Comment by beniwohli** [comment]
jenkins test this please

**Comment by bastbnl** [comment]
thx for the explanation @karmi. I think I made it

**Comment by karmi** [comment]
Great, @bastbnl!

**Comment by beniwohli** [comment]
Awesome work, @bastbnl, this looks great!

And thanks @karmi for helping with the logistics!

**Pull request merged**